packages/apputils/src/clientsession.tsx

```diff
@@ -739,6 +740,23 @@ class ClientSession implements IClientSession {
    * Handle a change to the session status.
    */
   private _onStatusChanged(): void {
+
+    // Set that this kernel is busy, if we haven't already
+    // If we have already, and now we aren't busy, dispose
+    // of the busy disposable.
+    if (this._setBusy) {
+      if (this.status === 'busy') {
+        if (!this._busyDisposable) {
+          this._busyDisposable = this._setBusy();
+        }
+      } else {
+        if (this._busyDisposable) {
+          this._busyDisposable.dispose();
+          delete this._busyDisposable;
```

Comment:
For example, when the session first starts there is no `_busyDisposable`, so if we get some status, we shouldn't try to dispose of it. We should only dispose of it when it exists.

Reply:
It should be safe to dispose of it twice, I think. But if we are worried about that, can we explicitly set it to `null` instead?

Reply:
Ah, I see what you mean. Yeah, I think it would be more consistent with the rest of the codebase to set the class member to `null` (we currently are only using the `delete` operator on plain `Object`s, rather than ES6 classes).

Reply:
Sure, just pushed.